src/app/components/Navbar.tsx

```diff
@@ -0,0 +1,173 @@
+        <div
+          className={` ml-0 flex justify-center md:justify-start flex-1 ${isMenuOpen ? "hidden" : ""}`}
+        >
+          <Image
+            src="/images/CURRENCI.png"
+            alt="Currenci Logo"
+            className="w-35 sm:w-30 md:w-36"
+            width={150}
+            height={150}
+          />
+        </div>
+
+        {/* Right Section (Desktop Buttons) */}
+        <div className="hidden md:flex items-center gap-3 lg:gap-6">
+          {menuLinks.map(({ name, href }) => (
+            <Link
+              key={name}
+              href={href}
+              className="text-xs  lg:text-lg font-semibold text-white hover:text-[#F2B263]"
+            >
+              {name}
+            </Link>
+          ))}
+          {buttons.map(({ name, href }) => (
+            <Link key={name} href={href}>
+              <div
+                className="px-3 py-1 sm:px-4 sm:py-2 border-2 border-[#F2B263] 
+               text-sm sm:text-xs  lg:text-base font-semibold 
+               rounded-lg sm:rounded-full 
+               text-white hover:bg-[#F2B263] hover:text-[#14342F] transition"
+              >
+                {name}
+              </div>
+            </Link>
+          ))}
+          {/* Login Button */}
+          <Link href="/login">
+            <div
+              className="px-3 py-1 sm:px-4 sm:py-2 border-2 border-[#F2B263] 
+        text-xs sm:text-xs  lg:text-base font-semibold 
+        rounded-lg sm:rounded-full 
+        text-white hover:bg-[#F2B263] hover:text-[#14342F] transition"
+            >
+              Login
+            </div>
+          </Link>
+        </div>
+      </div>
+      {/* Background Overlay when the menu is open */}
+      {isMenuOpen && (
+        <div
+          className="fixed inset-0 bg-black opacity-50 z-30"
+          onClick={toggleMenu} // Close menu when overlay is clicked
+          onKeyDown={(e) => {
+            if (e.key === "Enter" || e.key === " ") toggleMenu();
+          }}
+          role="button"
+          tabIndex={0}
+          aria-label="Close menu"
+        ></div>
+      )}
+
+      {/* Mobile Menu */}
+      <div
+        className={`md:hidden fixed top-0 left-0 w-[75%] h-full bg-[#1E3432] p-6 transition-all duration-300 z-40 transform ${
+          isMenuOpen ? "translate-x-0" : "-translate-x-full"
+        }`}
+      >
+        <div className="flex justify-between items-center mb-6">
+          {/* Close Icon */}
+          <div
+            className="cursor-pointer"
+            onClick={toggleMenu}
+            role="button"
+            tabIndex={0}
+            aria-label="Toggle menu"
+            onKeyUp={(e) =>
+              (e.key === "Enter" || e.key === " ") && toggleMenu()
+            }
+          >
+            <Icon
+              icon="mdi:close"
+              width={32}
+              height={32}
+              className="text-[#F2B263]"
+            />
+          </div>
+
+          {/* Logo */}
+          <Image
```

Comment:
can you put this image in the center of the navbar. Make the toggle button position absolute relative to the navbar